### ISSUE TITLE
show zipcode when looking at street adress

### ISSUE THREAD

**Issue by tjeulink** [osmandapp/OsmAnd#10717]
Currently i am unable to link an adress to a zip code simply by looking up the street adress. necessery for when writing mail.

**Comment by hans-fritz-pommes** [comment]
Yes, it's a OsmAnd specific problem, see related issues: #5705, #7874
